util.go

```diff
@@ -23,6 +23,23 @@ func isFalse(value interface{}) bool {
 	case nil:
 		return true
 	}
+	// Try the reflection cases before returning false.
+	vType := reflect.TypeOf(value).Kind()
```

Comment:
I think `reflect.ValueOf(value).Type()` and `reflect.TypeOf(value)` are equivalent here. If the reflect value is need below might want to switch this to ValueOf